translations/fr.json

```diff
@@ -260,8 +255,7 @@
   "building.names.ice_bank": "Banquise",
   "building.names.iceport": "Avant-poste",
   "building.names.icetemple": "Temple de glace",
-
-  "unit": "",
+  "unit": "unit",
```

Comment:
unit have two problems: it's in english and in the french game, it makes no sense. This word is use with the name of the unit like "swordman unit" and in french we say "épéiste" not "épéiste unité"